virtual_rainforest/core/data.py

```diff
@@ -364,7 +364,7 @@ def add_from_dict(self, output_dict: Dict[str, DataArray]) -> None:
         """
 
         for variable in output_dict:
-            self.data[variable] = output_dict[variable]
+            self[variable] = output_dict[variable]
```

Comment:
Why is the variable assigned to the `model` rather than `model.data `?

Reply:
`self.data` selects the dictionary called `data` stored within the `Data` object, which can be updated directly.

`self` selects the `Data` object, which updates via its `__setitem__` method. This is a method that David wrote that does some sanity checking of the updated data (i.e. saving only `DataArray`), and gives `LOGGER` output for whether a variable is being updated or added.

This method is the main source of the logger output that floods the screen when you run `vr_run`, so it might need to be altered in future, but it's probably sensible to ensure that every update passes through this function for consistencies sake.